test/sagemaker_tests/pytorch/training/integration/sagemaker/test_pt_s3_plugin.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import boto3
+import pytest
+from sagemaker import utils
+from ...integration import (DEFAULT_TIMEOUT, resnet18_path)
+from ...integration.sagemaker.timeout import timeout
+
+MULTI_GPU_INSTANCE = 'ml.p3.8xlarge'
+RESOURCE_PATH = os.path.join(os.path.dirname(__file__), '..', '..', 'resources')
+
+
+@pytest.mark.processor("gpu")
+@pytest.mark.integration("pt_s3_plugin")
+@pytest.mark.model("resnet18")
+@pytest.mark.skip_cpu
+@pytest.mark.skip_py2_containers
+def test_pt_s3_plugin_sm_gpu(sagemaker_session, framework_version, ecr_image):
```

Comment:
Is there an equivalent test for CPU DLCs as well?

Since there is a minimum allowed framework version below which this test will fail on DLC images, could you insert something similar to [can_run_dataparallel](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smdataparallel.py#L32) for this test?